.gitignore

```diff
@@ -48,3 +48,5 @@ ramenctl/build
 # Generated disk images
 *.qcow2
 *.iso
+
+config/crd/bases/_.yaml
```

Comment:
sure, but this dummy file is always created when command `make manifests` is executed.

Reply:
Yes, that's correct. I don't know why that file is being created either. Maybe just a bug in the compiler version we are using.

Reply:
The root cause is here: https://github.com/RamenDR/ramen/pull/1418#discussion_r1649688011

IOW commit https://github.com/RamenDR/ramen/commit/85fa41c3f9b57a62d94b966a9189ea740334ddb2 introduced this, but unsure of the actual fix in this case.

I would not add it to git ignore so that we fix the problem at some point in time.